README.md

```diff
@@ -70,6 +72,8 @@ format: String, image format, 'png' or 'jpg', Default is 'png'
 
 processor: String, css processor, 'css' or 'less' or 'sass' or 'scss', Default is 'css'
 
+retina: Boolean, returns higher quality image, Default is false
```

Comment:
Nice idea; I guess the default could be 1, right? I like `dotsPerPixel` better than `dppx` myself.